src/Test/Utilities/Portable/ModuleInitializer.cs

```diff
@@ -15,6 +16,9 @@ internal static void Initialize()
         {
             Trace.Listeners.Clear();
             Trace.Listeners.Add(new ThrowingTraceListener());
+
+            FatalError.Handler = e => Assert.False(true, $"Fatal exception: {e}");
+            FatalError.NonFatalHandler = e => Assert.False(true, $"Non-fatal exception: {e}");
```

Comment:
💡 Calling `Debug.Fail` instead of `Assert.False` will force this through the existing trace listener.